### ISSUE TITLE
Gallery app only shows the image and no controls.

### ISSUE THREAD

**Issue by EnterTheVoid-x86** [Alex313031/ThoriumOS#30]
![image](https://github.com/Alex313031/ThoriumOS/assets/64978924/c5baac6d-5101-4de7-92d9-a3a955e51b90)

I haven't tried a video yet but this is the only thing I see when opening images.

**Comment by Alex313031** [comment]
@EnterTheVoid-x86 Yeah.... This is google being a dick. The code for the gallery app is closed source, so this is basically just a shim/stub app with only the bare minimum code needed to open a video or image. Stuff doesn't scale to fit the window, and there are no controls or any of the extra stuff you see in ChromeOS proper. There's really not much I can do. I recommend using https://chromewebstore.google.com/detail/mediaplayer-video-and-aud/mgmhnaapafpejpkhdhijgkljhpcpecpj
